### DISCUSSION TITLE
增加抽奖功能,就是那种转评赞的

### DISCUSSION THREAD

**Discussion by michaellongge163** [the1812/Bilibili-Evolved#3190] [功能建议 / Ideas]
wwwwwwwww

**Comment by Geeyun-JY3** [comment]
啥意思，说清楚，自动赞转评抽奖动态？

**Reply by michaellongge163** [reply]
对

**Reply by michaellongge163** [reply]
很多抽奖需要关注2个人,比如这样的,但是你又一般没有关注第二个人,所以最好能自动关注第二个人,然后抽奖,顺便把第二个人的动态也抽了
[[#小芙福利站#]小芙开始发福利啦！章鱼娘也来加码啦！！关注 [@芙清FulQun ](https://space.bilibili.com/475014368/dynamic)+[@聚划算 ](https://space.bilibili.com/286980475/dynamic)，转发+评论该条动态，截止至4月6日，我会在评论区抽一条小锦鲤获得200元红包哦[#互动抽奖#]]

**Comment by the1812** [comment]
这不成助长抽奖号的工具了吗

**Comment by michaellongge163** [comment]
对,自动转评赞










在 2022-03-28 11:31:30，"Geeyun-JY3" ***@***.***> 写道：

啥意思，自动赞转评抽奖动态？

**Reply by Geeyun-JY3** [reply]
不用再回了，你这需求已经被拒了。
